L1Trigger/DTTrigger/src/DTTrigTest.cc

```diff
@@ -58,6 +58,7 @@ DTTrigTest::DTTrigTest(const ParameterSet& pset) : my_trig(nullptr) {
   my_params = pset;
   if (my_debug)
     cout << "[DTTrigTest] Constructor executed!!!" << endl;
+  my_trig = new DTTrig(my_params, consumesCollector());
```

Comment:
Why have you moved this line?

Reply:
Because the consumesCollector() is only supposed to be called in the constructor. Calling it in the `beginRun` is forbidden.